lib/src/kbasesearchengine/search/ElasticIndexingStorage.java

```diff
@@ -331,21 +331,26 @@ public void flushIndexing(SearchObjectType objectType) throws IOException {
     
```

Comment:
Things would probably be even more readable if we used ImmutableMap.of() calls to build the maps in a nested fashion rather than declaring each map as a top level variable. I wish java had map and list literals but ImmutableMap is the closest I've found so far. Again, not blocking this PR - getting rid of the anon classes is great.